### DISCUSSION TITLE
Downloading ini files

### DISCUSSION THREAD

**Discussion by Irfartz** [jserodio/rg351p-ppsspp-settings#20] [Q&A - Drop your questions here]
Hi im new to emulation and the rg351p is my first emulation device. i was wondering as to how i can download these ini files for ppsspp 
 games cuz this is all very new to me and if there are any more steps i need to follow after downloading these files. I hope im not asking for too much.

**Comment by jserodio** [comment]
hi @Irfartz,
I don't really have any psp games at the moment in my SDs so I can't give you the exact details.
1- boot your game
2- go to ppsspp settings
3- create game config
4- locate the .ini file your game has created within the ppsspp folder (I don't remember the exact path)
5- locate the .ini of the game here in github, edit, select everything but the code, and paste it in the one your ppsspp folder
I hope that helps,
